.travis.yml.jinja

```diff
@@ -0,0 +1,82 @@
+language: python
+cache:
+  directories:
+    - $HOME/.cache
+
+python:
+  {#- Pin the lowest supported version for each Odoo release #}
+  - "3.6"
+
+addons:
+  {#- Pin the lowest supported version for each Odoo release #}
+  postgresql: "9.6"
+  apt:
+    {%- if travis_apt_sources %}
+    sources:
+      {%- for source in travis_apt_sources %}
+      - {{ source|tojson }}
+      {%- endfor %}
+    {%- endif %}
+    packages:
+      - expect-dev # provides unbuffer utility
+      {%- for package in travis_apt_packages %}
+      - {{ package|tojson }}
+      {%- endfor %}
+
+stages:
+  - linting
+  - test
+
+jobs:
+  include:
+    - stage: linting
+      name: "pre-commit"
+      install: pip install pre-commit
+      script: pre-commit run --all --show-diff-on-failure --verbose --color always
+      after_success:
+      before_install:
+    {%- if rebel_module_groups %}
+    {#- If there are rebel modules, we need separated test jobs and .pot generations #}
+    {%- for group in rebel_module_groups %}
+    # Test separately: {{ group }}
+    - stage: test
+      env:
+        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT=1 INCLUDE="{{ group }}"
+    - stage: test
+      env:
+        - TESTS=1 ODOO_REPO="OCA/OCB" INCLUDE="{{ group }}"
+    {%- endfor %}
+    # Test all other addons together
+    - stage: test
+      env:
+        - TESTS=1 ODOO_REPO="odoo/odoo" EXCLUDE="{{ rebel_module_groups|join(',') }}"
+    - stage: test
+      env:
+        - TESTS=1 ODOO_REPO="OCA/OCB" EXCLUDE="{{ rebel_module_groups|join(',') }}"
+    {%- else %}
+    {#- If all modules can get along, we test and generate .pot all at once #}
+    - stage: test
+      env:
+        - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+    - stage: test
+      env:
+        - TESTS="1" ODOO_REPO="OCA/OCB"
+    {%- endif %}
+env:
+  global:
+    - VERSION="{{ odoo_version }}" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
```

Comment:
MQT now has an option to [install dependencies with pip](https://github.com/OCA/maintainer-quality-tools/blob/67d3880b085a99d820e5a65cc7354dae95d1b4b0/travis/travis_install_nightly#L119-L134) (env var `MQT_DEP="PIP"`). We are experimenting that in some repos. Can it be part of the template?

Reply:
Yes, of course. Let me check that.